torch/nn/modules/flatten.py

```diff
@@ -31,3 +34,83 @@ def __init__(self, start_dim: int = 1, end_dim: int = -1) -> None:
 
     def forward(self, input: Tensor) -> Tensor:
         return input.flatten(self.start_dim, self.end_dim)
+
+    def extra_repr(self) -> str:
+        return 'start_dim={}, end_dim={}'.format(
+            self.start_dim, self.end_dim
+        )
+
+
+class Unflatten(Module):
+    r"""
+    Unflattens a tensor into another tensor of shape (N, C, H, W). For use with :class:`~nn.Sequential`.
```

Comment:
I think you need to update this part of the doc to mention the two ways to call this.
The two examples below are very good!